nautobot_ui/README.md

```diff
@@ -0,0 +1,29 @@
+# Nautobot v2 UI Prototype
+
+## Running it
+
+Install and compile dependencies:
+
+```no-highlight
+nautobot-server build
+```
+
+Configure Nautobot to pass through authentication over CORS by putting this into your `nautobot_config.py`:
+
+```python
+CORS_ALLOW_ALL_ORIGINS = True
+CORS_ALLOW_CREDENTIALS = True
+SESSION_COOKIE_SAMESITE = None
+```
+
+Set the `NAUTOBOT_API_TOKEN` prior to starting Node.js:
+
+```no-highlight
+export NAUTOBOT_API_TOKEN=nnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnn
```

Comment:
Presumably you need to actually create this API token as well... wonder if we should override `createsuperuser` to give the option to provision a token too?

Reply:
I'm actually thinking we should have a `Token` created by default for any new user. Simplify.

Reply:
Hmm, not sure I'd agree with that - especially if we are wanting to change things so that an admin cannot retrieve/view the tokens of other users, how would the newly created token get communicated to the user in a more typical workflow?